src/components/Wrappers/withClaimReturned.js

```diff
@@ -15,6 +66,180 @@ const withClaimReturned = WrappedComponent => class withClaimReturnedComponent e
     };
   }
 
+  refundTransfers = async () => {
+    const getAccounts = (itemBarcode, userId) => {
+      const {
+        mutator: {
+          accounts: {
+            GET,
+          },
+        }
+      } = this.props;
+      const lostStatus = 'Lost item fee';
+      const processingStatus = 'Lost item processing fee';
+      const pathParts = [
+        'accounts?query=',
+        `barcode=="${itemBarcode}"`,
+        ` and userId=="${userId}"`,
+        ` and (feeFineType=="${lostStatus}"`,
+        ` or feeFineType=="${processingStatus}")`
+      ];
+      const path = pathParts.reduce((acc, val) => acc + val, '');
```

Comment:
nothing wrong with `reduce`, but a simpler approach is ```const path = pathParts.join('');```

Reply:
If necessary we can change it